contracts/ClaimsManager.sol

```diff
@@ -287,6 +292,51 @@ contract ClaimsManager is
         );
     }
 
+    function downgradePolicy(
+        address claimant,
+        address beneficiary,
+        uint256 coverageAmountInUsd,
+        uint256 claimsAllowedFrom,
+        uint256 claimsAllowedUntil,
+        string calldata policy,
+        string calldata metadata
+    ) external override returns (bytes32 policyHash) {
+        require(claimant == msg.sender, "Sender not claimant");
+        policyHash = keccak256(
+            abi.encodePacked(
+                claimant,
+                beneficiary,
+                claimsAllowedFrom,
+                policy,
+                metadata
+            )
+        );
+        PolicyState storage policyState = policyHashToState[policyHash];
+        require(policyState.claimsAllowedUntil != 0, "Policy does not exist");
+        require(
+            policyState.coverageAmountInUsd >= coverageAmountInUsd,
+            "Increases coverage amount"
+        );
+        require(
+            policyState.claimsAllowedUntil >= claimsAllowedUntil,
```

Comment:
Should we also validate that `claimsAllowedUntil` is not before `claimsAllowedFrom`?

Reply:
I might be missing something here but don't we have to force `claimsAllowedUntil` to be equal to `policyState.claimsAllowedUntil`?

Reply:
Downgrade here means reducing coverage and/or reducing the duration (so it's the exact opposite of upgradePolicy)

Reply:
Fixed in https://github.com/api3dao/claims-manager/pull/55/commits/3391f023a56a43c649e8ce060f103aaed60acf74